### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by yashodhicy**
🙆🏻‍♂️ Add any missing JSX elements to project (without adding extra functionality)
🙆🏻‍♂️  final design  match zeplin's design
🙆🏻‍♂️  use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,39 +1,76 @@
...
+          <h4 className={styles.genre}>fiction</h4>
+          <input
+            type="text"
+            className={styles.title}
+            value={bookProp[0].title}
+            readOnly
+          />
+          <h4 className={styles.author}>{bookProp[0].author}</h4>
+          <div className={styles.actions}>
+            <ul className={styles.actionsul}>
```
Comment (GzimAsani) [inline src/components/Book.js:23]: - [x] Remove spaces here in order to match better the design 
image

File: src/App.css [hunk 2]
```diff
@@ -1,3 +1,111 @@
...
+#root {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.body {
+  display: flex;
```
Comment (GzimAsani) [inline src/App.css]: - [x] You have used display flex many times which means you are repeating yourself and this falls under the category of repeating yourself which isn't recommended and is considered bad css practice, I would suggest you create a class with dispaly: flex inside of it and then use it whenever you need on your html 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @yashodhicy                    :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the styling is 99% done +1: 

 ## Required Changes 

- [x] Check for the comments :arrow_double_down: 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @yashodhicy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**